.github/workflows/docs-changelog.yaml

```
@@ -69,67 +60,35 @@ jobs:
-      - name: Update reference
-        run: |
-          REFERENCE="${{ inputs.reference }}"
-          NAME="${REFERENCE#deploy/}"
-          echo "NAME=$NAME" >> $GITHUB_ENV
-
-      - name: Create the changelog
+      - name: Create the changelog file
         working-directory: fa/changelog/snippets # script assumes it is in this directory
+        # We need to remove the `/` any anything in front of it from the branch name.
+        # Specifically the `/` is problematic in a file name.
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:22: Double quote to prevent globbing and word splitting

(shellcheck)